specification/graphrbac/data-plane/stable/1.6/graphrbac.json

```diff
@@ -1592,6 +1592,75 @@
           }
         }
       }
+    },
+    "/{tenantID}/oauth2PermissionGrants" : {
+      "get" : {
+       "tags": [
+         "OAuth2"
+       ],
+        "operationId": "OAuth2_Get",
+        "description": "Queries OAuth2 permissions for the relevant SP ObjectId of an app.",
+        "produces" : [ "application/json" ],
+        "parameters" : [
+        {
+          "name" : "$filter",
+          "in" : "query",
+          "required" : false,
+          "type" : "string",
+          "x-example" : "clientId+eq+'61ed44c3-5a1d-4639-a215-07f25129c6c3'"
```

Comment:
The linter is complaining about not having a "description" property here.

Reply:
Added a description